gluten-core/src/main/scala/io/glutenproject/extension/columnar/TransformHintRule.scala

```diff
@@ -313,8 +313,8 @@ case class AddTransformHintRule() extends Rule[SparkPlan] {
   private def addTransformableTag(plan: SparkPlan): Unit = {
     if (TransformHints.isAlreadyTagged(plan)) {
       logDebug(
-        s"Skipping executing" +
-          s"io.glutenproject.extension.columnar.CheckTransformableRule.addTransformableTag " +
+        s"Skipping execute " +
+          s"io.glutenproject.extension.columnar.AddTransformHintRule.addTransformableTag " +
```

Comment:
Maybe, leaving some brief & readable message is enough. Right?
E.g., "Skip adding transformable tag, since ...".

Reply:
ok